extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -217,7 +218,7 @@ public function getImageSizesToDisplay( $thumbnail ) {
 			$height = $thumbnail->getHeight();
 		}
 
-		return [ 'height' => $height, 'width' => $width ];
+		return [ 'height' => round( $height ), 'width' => round( $width ) ];
```

Comment:
I'm still wondering about whether it's best as round, ceil or floor - what are the grounds for using round?